README.md

```diff
@@ -100,6 +101,18 @@ kubectl create namespace greeting-workflow
 kubectl apply -f config/samples/sw.kogito.kie.org__v08_kogitoserverlessworkflow.yaml -n greeting-workflow
 ```
 
+8. Default configuration
+
+A configmap called kogito-builder-config will be created under the kogito-builder namespace at the first application of a workflow, it contains:
+  
+- BUILDER_RESOURCE_DEFAULT = Dockerfile
+- WORKFLOW_DEFAULT_EXTENSION = .sw.json
+- DEFAULT_KANIKO_SECRET_DEFAULT = regcred
+- DEFAULT_REGISTRY_REPO =  quay.io/kiegroup
+- Dockerfile = dockerfile content
+
+For the local development the DEFAULT_REGISTRY_REPO must be changed
+
 ## Contributing
```

Comment:
would be nice to break this in another .md file like we can find on another projects.

Reply:
When will be merged with the platform